packages/@lwc/wire-service/src/wiring.ts

```diff
@@ -168,18 +168,36 @@ export class WireEventTarget {
 
                 // setup listeners for all reactive parameters
                 const configContext = this._context[CONTEXT_ID][CONTEXT_UPDATED];
+                const reactiveParametersGroupByHead = new Map<string, Set<ReactiveParameter>>();
```

Comment:
Why not using a Set instead of an Array? Each item is guaranteed to be unique because the `buildReactiveParameter` creates a brand new object every time.

Reply:
Also `Map` has poor performance on IE11. Any reason why not using an `Object` instead of a `Map`?

Reply:
The Map, no particular reason, will change.
The Set, because of the `.mutated` is a set, but you are right, for this case there is no need for it to be a set.